automerge-cli/src/export.rs

```diff
@@ -0,0 +1,42 @@
+use crate::error::AutomergeCliError;
+use automerge_backend::AutomergeError;
+use automerge_frontend::Value;
+use automerge_protocol::Patch;
+use std::fs::File;
+use std::path::Path;
+use std::{io, io::Read};
+
+fn get_changes(changes_file: &Path) -> io::Result<Vec<u8>> {
+    let mut input_data = Vec::new();
+    let mut input_file = File::open(changes_file)?;
+    input_file.read_to_end(&mut input_data)?;
+
+    Ok(input_data)
+}
+
+fn get_patch(changes_bytes: Vec<u8>) -> Result<Patch, AutomergeError> {
+    let mut backend = automerge_backend::Backend::init();
+    backend.load_changes_binary(vec![changes_bytes]).unwrap();
```

Comment:
`load_changes_binary` followed by `get_patch` is semantically equivalent to `apply_changes_binary` and slower